src/Host/Client/Test/ExecutionTracing/BreakpointsTest.cs

```diff
@@ -21,11 +18,12 @@ namespace Microsoft.R.ExecutionTracing.Test {
     [ExcludeFromCodeCoverage]
     [Category.R.ExecutionTracing]
     public class BreakpointsTest : IAsyncLifetime {
+        private readonly TestCoreShell _coreShell = new TestCoreShell(null);
```

Comment:
The reason why `CoreServicesFixture` was introduced in the first place is to support logging in tests and store them as test artifacts. It can be easily converted into `ServiceManagerFixture`.